Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```diff
@@ -392,3 +393,26 @@ FRFSRecon:
     settlementAmount: (.amount)|I
     differenceAmount: Kernel.Prelude.fmap (.amount)|I
     currency: (Kernel.Prelude.Just . (.currency))|I
+
+FRFSConfig:
+  tableName: frfs_config
+  derives: "Generic,Show"
+  fields:
+    merchantOperatingCityId: Id MerchantOperatingCity
+    merchantId: Id Merchant
+    bookingStartTime: UTCTime
+    bookingEndTime: UTCTime
+    routndTripTicketLimit: Int
+    oneWayTicketLimit: Int
+    city: City
```

Comment:
There is `merchantOperatingCityId`, do you need `city`?

Reply:
yes for frontend we are fetching the data using city